cluster/gce/gci/configure-helper.sh

```diff
@@ -706,9 +705,6 @@ function start-kube-addons {
   fi
   if [[ "${ENABLE_L7_LOADBALANCING:-}" == "glbc" ]]; then
     setup-addon-manifests "addons" "cluster-loadbalancing/glbc"
```

Comment:
but then it won't start-lb-controller without the right env vars, which makes the function name a lie right?